src/core/server/status/plugins_status.ts

```diff
@@ -76,6 +76,7 @@ export class PluginsStatusService {
 
   public getDerivedStatus$(plugin: PluginName): Observable<ServiceStatus> {
     return this.update$.pipe(
+      debounceTime(25),
```

Comment:
If we add a debounce here, do we still need it above? https://github.com/elastic/kibana/blob/524401973f1d0ac5403cd0fbb3ea82a63962a45a/src/core/server/status/plugins_status.ts#L72-L73

Reply:
nit: Would be helpful to copy that comment here too for clarity

Reply:
I agree! Added a comment to the new debounceTime usage :)